tools/build.sh

```diff
@@ -1,12 +1,5 @@
 #!/bin/bash
 
```

Comment:
I think this script should look for a `.env` file which includes the `export ALLOWED_ORIGIN = ... `.

Since we are building it only for deployments, our default config should even be part of the repo, I'd argue. But thus easily changeable by the community.

Reply:
I'm thinking the script should load the testnet config by default, and if run with `--mainnet` flag, it loads the mainnet config.

Reply:
I think setting all variables per environment is the most flexible way, especially useful for deploying any combination of our apps' versions for testing purposes. So a dedicated config file would be just unnecessary overhead. That's more or less the feedback I got for https://github.com/nimiq/accounts/pull/32 (would also ready for a new shot soon which the same approach).